src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,8 @@
+*,
+*:before,
+*:after {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+}
```

Comment:
It is not the optimal solution. Not use * for clear geometry (padding, margin...) for elements